client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -133,6 +129,21 @@ class EmailProvidersComparison extends Component {
 		this.isMounted = false;
 	}
 
+	expandedStateProvider( source ) {
```

Comment:
Naming suggestion: might it be more explicit to call this `getDefaultExpandedState( source )`?